src/Library/demos/Spin Button/main.js

```diff
@@ -0,0 +1,48 @@
+import Gtk from "gi://Gtk?version=4.0";
+import GObject from "gi://GObject";
+
+Gtk.init();
+
+const open_spin_box = workbench.builder.get_object("open_spinbox");
+
+// Create new Gtk instances
+const spin_button = new Gtk.SpinButton();
+const adjustment = new Gtk.Adjustment({
+  lower: 0,
+  page_increment: 1,
+  step_increment: 1,
+  upper: 100,
+  value: 0,
+});
+
+spin_button.set_adjustment(adjustment);
```

Comment:
Why in a separate window? Please move directly to the `main.blp` so it's clear how to use with Blueprint and so that there is no need to click a button.

There is already an example in Workbench on how to use adjustment with blueprint.

Reply:
That means I do not need to create a _.js_ file since the spin button is created with blueprint, right?

Reply:
we still need the `.js` file to demonstrate the APIs

Perhaps the JavaScript could log the value when it changes?